packages/core/src/external/fhir/bundle/qa.ts

```diff
@@ -17,7 +18,30 @@ export function checkBundleForPatient(
   patientId: string
 ): true {
   const { log } = out(`checkBundleForPatient - cx ${cxId}, pat ${patientId}`);
+  const additionalInfo = { cxId, patientId };
 
+  // check there's one and only one patient
```

Comment:
these comments feel not necessary -- it's very clear.

Reply:
done